src/modules/displaymanager/main.py

```diff
@@ -147,19 +176,40 @@ def set_autologin(username, displaymanagers, default_desktop_environment, root_m
                 else:
                     gdm_conf.write('AutomaticLoginEnable=False\n')
 
-        if do_autologin and os.path.exists("{!s}/var/lib/AccountsService/users".format(root_mount_point)):
-            os.system("echo \"[User]\" > {!s}/var/lib/AccountsService/users/{!s}".format(root_mount_point, username))
+        if (do_autologin
+                and os.path.exists("{!s}/var/lib/AccountsService/users".format(
+                    root_mount_point
+                    )
+                )):
+            os.system(
+                "echo \"[User]\" > "
+                + "{!s}/var/lib/AccountsService/users/{!s}".format(
+                    root_mount_point,
+                    username
+                    )
+                )
 
             if default_desktop_environment is not None:
-                os.system("echo \"XSession={!s}\" >> {!s}/var/lib/AccountsService/users/{!s}".format(
-                          default_desktop_environment.desktop_file, root_mount_point, username))
-
-            os.system("echo \"Icon=\" >> {!s}/var/lib/AccountsService/users/{!s}".format(
-                      root_mount_point, username))
+                os.system(
+                    "echo \"XSession={!s}\" >> "
+                    + "{!s}/var/lib/AccountsService/users/{!s}".format(
+                        default_desktop_environment.desktop_file,
+                        root_mount_point, username
+                        )
+                    )
+
+            os.system(
+                "echo \"Icon=\" >> "
+                + "{!s}/var/lib/AccountsService/users/{!s}".format(
+                    root_mount_point, username
+                    )
+                )
 
     if "kdm" in displaymanagers:
         # Systems with KDM as Desktop Manager
-        kdm_conf_path = os.path.join(root_mount_point, "usr/share/config/kdm/kdmrc")
+        kdm_conf_path = os.path.join(
```

Comment:
Given that all the other path manipulations are just doing string concatenation UNIX-style, I'm surprised to see os.path.join here. Not your code, and let's leave it, but there's some consistency missing.